internal/storage/v2/clickhouse/schema/schema.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE IF NOT EXISTS spans (
+    id String,
+    trace_id String,
+    trace_state String,
+    parent_span_id String,
+    name String,
+    kind String,
+    start_time DateTime64(9),
+    status_code String,
+    status_message String,
+    duration Int64,
+
+    events Nested (
+        name String,
+        timestamp DateTime64(9)
+    ),
+
+    
+    links Nested (
+        trace_id String,
+        span_id String,
+        trace_state String
+    ),
+
+    service_name String,
+
+    scope_name String,
+    scope_version String
+)
+ENGINE = MergeTree
+PRIMARY KEY(trace_id);
```

Comment:
should we add any other primary keys here?

Reply:
@yurishkuro Primary keys in ClickHouse work differently and are not required to be unique per row. The primary key instead acts like an index to reduce the amount of data that is scanned (more info at https://clickhouse.com/docs/migrations/postgresql/data-modeling-techniques#how-are-clickhouse-primary-keys-different).